scripts/examples/bash-cli/README

```diff
@@ -1,49 +1,28 @@
-
-Introduction
-
-   This file describes the XE CLI + bash examples included in the [1]Software
-   Development Kit (SDK) for the [2]XenServer 4.0 product family.
-
 XE CLI + bash examples
 
-  install-debian
-
-   [3][ browse ] [4][ download ]
+ [install-debian][1]
 
    A very simple example which installs Debian Etch 4.0 from the pre-existing
    VM template.
 
-  suspend-resume
-
-   [5][ browse ] [6][ download ]
+[suspend-resume][2]
 
    A very simple example which demonstrates how to suspend and resume a VM.
 
-  clone-vms
-
-   [7][ browse ] [8][ download ]
+[clone-vms][3]
 
    An example which takes the uuid of a VM which is either running or halted.
    If it is running it is shutdown. The VM is cloned and if the original VM was
    running then both it and the clone are started.
 
-  move-management-to-bond
-
-   [9][ browse ] [10][ download ]
+[move-management-to-bond][4]
 
    An example which detects when some hosts management interfaces are still
    using a regular interface when a suitable bond configuration exists, and
    moves the management interfaces to the bond.
 
-References
-
-   1. file://localhost/bind/myrepos/xen-api/scripts/index.html
-   2. http://www.xensource.com/
-   3. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/install-debian.html
-   4. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/install-debian
-   5. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/suspend-resume.html
-   6. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/suspend-resume
-   7. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/clone-vms.html
-   8. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/clone-vms
-   9. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/move-management-to-bond.html
-  10. file://localhost/bind/myrepos/xen-api/scripts/examples/bash-cli/move-management-to-bond
+
+[1]: install-debian
```

Comment:
Apologies, I don't understand your question. Are you asking whether this particular example is useful or whether it's useful to have a link to the example in the readme file?